src/collectors/monit/monit.py

```diff
@@ -40,12 +42,19 @@ def get_default_config(self):
             'path':         'monit',
             'byte_unit':    ['byte'],
             'send_totals':  False,
+            'scheme':       'http',
+            'selfsigned':   False,
         })
         return config
 
     def collect(self):
-        url = 'http://%s:%i/_status?format=xml' % (self.config['host'],
+        url = '%s://%s:%i/_status?format=xml' % (self.config['scheme'],
+                                                 self.config['host'],
                                                    int(self.config['port']))
+
+        if self.config['selfsigned'] and sys.hexversion >= 0x020709f0 and hasattr(ssl, '_create_unverified_context'):
```

Comment:
Can you add a comment as to what version  `0x020709f0` correlates to? or use `sys.version_info` instead?